src/main/java/org/openstreetmap/atlas/checks/utility/ElevationUtilities.java

```diff
@@ -0,0 +1,361 @@
+package org.openstreetmap.atlas.checks.utility;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Serializable;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.openstreetmap.atlas.geography.Location;
+import org.openstreetmap.atlas.geography.Longitude;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+import org.openstreetmap.atlas.utilities.scalars.Distance;
+
+/**
+ * Used to determine assumed direction for some features (e.g. waterways). Assumptions are that any
+ * SRTM file follows the specification here:
+ * https://dds.cr.usgs.gov/srtm/version2_1/Documentation/Quickstart.pdf
+ *
+ * @author Taylor Smock
+ */
+public final class ElevationUtilities implements Serializable
+{
+    private static final long serialVersionUID = -5929570973909280629L;
+    /**
+     * The assumed file extension
+     */
+    private static final String SRTM_EXT = "hgt";
```

Comment:
Nit:
```suggestion
    private static final String SRTM_EXTENTION = "hgt";
```

Reply:
I've expanded EXT/Ext to the appropriately cased extension.